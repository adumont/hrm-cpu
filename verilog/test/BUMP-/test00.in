22 b6
46 a6
55 12
32 da
58 eb
39 a5
19 c9
15 f6
41 22
16 4d
55 ec
39 ee
38 91
28 36
22 de
33 34
25 85
28 96
17 ca
30 ad
38 c5
51 09
03 c9
32 91
