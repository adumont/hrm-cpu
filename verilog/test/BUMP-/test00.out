b5
a5
11
d9
ea
a4
c8
f5
21
4c
eb
ed
90
35
dd
33
84
95
c9
ac
c4
08
c8
90
